cosmic-settings/src/pages/desktop/window_management.rs

```diff
@@ -132,6 +146,18 @@ impl Page {
                     error!(?err, "Failed to set config 'cursor_follows_focus'");
                 }
             }
+            Message::ShowMaximizeButton(value) => {
+                cosmic::config::COSMIC_TK.write().unwrap().show_maximize = value;
```

Comment:
You'll need to write the config to disk as well with its cosmic_config context

Reply:
Done